src/projections/gui/RangeDialog.java

```diff
@@ -453,12 +453,14 @@ private boolean isInputValid(){
 			return false;
 		}
 
+		/*
```

Comment:
Maybe add a comment clarifying the use case of setting the end time past the total time here